packages/canonical-path/src/platform/content.js

```diff
@@ -18,9 +18,22 @@ const pathResolvers = {
       'site.$id': site.id(),
     };
     const section = await load('websiteSection', id, { alias: 1 }, query);
+    if (section) return section.alias;
     // @todo This should eventually account for secondary sites/sections.
-    // For now load home when not found.
-    return section ? section.alias : 'home';
+    // For now load an alternate from schedules
+    const sectionQuery = getAsArray(content, 'sectionQuery');
+    if (sectionQuery.length) {
+      const currentSiteSections = sectionQuery.map((schedule) => {
+        if (schedule.siteId === site.id()) return schedule;
+        return null;
+      }).filter(v => v);
+      if (currentSiteSections.length) {
+        const currentSiteSection = await load('websiteSection', currentSiteSections[0].sectionId, { alias: 1 }, query);
+        if (currentSiteSection) return currentSiteSection.alias;
+      }
+    }
```

Comment:
Use `Array.find` instead of map+filter when returning a single result.
```suggestion
    const found = sectionQuery.find(({ siteId }) => siteId === site.id());
    if (found) {
      const secondary = await load('websiteSection', found.sectionId, { alias: 1 }, query);
      if (secondary) return secondary.alias;
    }
```